src/main/java/net/masterthought/cucumber/json/Match.java

```diff
@@ -1,13 +1,23 @@
 package net.masterthought.cucumber.json;
 
+import com.google.gson.JsonElement;
+
 public class Match {
 
     // Start: attributes from JSON file report
-    private final String location = null;
+    private final JsonElement location = null;
     // End: attributes from JSON file report
 
     public String getLocation() {
-        return location;
+        if (location.isJsonPrimitive() && location.getAsJsonPrimitive().isString()) {
```

Comment:
No, it will throw npe, if match doesnt have location. 
We could possible set it to empty string, but i don't how it would affect Step Page.